tests/distributed/test_common.py

```
@@ -108,10 +110,11 @@ def move_params_to_cpu(parameters):
 @is_trainium_test
 class TestCommonDistributed(DistributedTest):
     # TODO: enable dp=4,tp=pp=2 when working on the multi-node training PR.
+    # Anything related to PP seems broken because of the way the model is traced.
```

If PP is broken, I think it is better to raise an error. Remove the commented combination, and/or isolate it in another test that checks that en error is raised when PP is used.